news/_posts/2022-06-06-prestashop-feature-freeze-v8.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "PrestaShop Version 8.0 Feature Freeze: Important Features"
+subtitle: "Feature Freeze: an Inside Look at Most Important Features"
+date:   2022-06-06 09:10:11
+authors: [ PrestaShop ]
+icon: icon-compass
+tags:
+ - news
+ - version
+ - 8.0
+---
+
+Dear developers and contributors,
+
+We are very pleased to be announcing the feature freeze of PrestaShop version 8.0. In this post, we will be looking at the most important features of this feature freeze, letting you know why we have frozen new features and explain what this means for the next few months.
+
+### Feature freeze is now effective
+
+Therefore, as of today, the feature freeze is now effective and no new features will be added to version 8.0, so we can work on polishing the 8.0 features and working on their quality.
+
+In the next few months, we will still be welcoming new features. We will be creating the 8.0.x branch to stabilize the code of the next 8.0 and new features will be added on “develop”.
+
+### Most important features of version 8.0
+
+In this section, we wanted to highlight the most important features of this version 8.0 feature freeze, visible below:
+1. Update to[ Symfony 4.4](https://symfony.com/releases/4.4) (LTS version, support until November 23)
+2. Compatibility with[ PHP 8](https://www.php.net/) / 8.1
```

Comment:
I think we can just say PHP 8.1 as it implicitly means that it is compatible with PHP 8.0 too?
```suggestion
1. Update to [Symfony 4.4](https://symfony.com/releases/4.4) (LTS version, support until November 23)
2. Compatibility with [PHP 8.1](https://www.php.net/)
```